tests/test_enrollment_api.py

```diff
@@ -55,3 +57,8 @@ def test_enroll_user_in_course():
     assert actual_response == expected_response
     request = responses.calls[0][0]  # pylint: disable=no-member
     assert json.loads(request.body) == expected_response
+
+
+def test_enroll_locally_raises():
+    with raises(NotConnectedToEdX):
```

Comment:
@haikuginger and what about happy path, when it should actually work? Genuine question - is there a test that tests that?